specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/apimanagement.json

```diff
@@ -0,0 +1,835 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ApiManagementClient",
+    "description": "Use these REST APIs for performing operations on entities like API, Product, and Subscription associated with your Azure API Management deployment.",
+    "version": "2022-08-01"
```

Comment:
>2022 [](http://example.com/codeflow?start=16&length=4)

Also please address all the breaking changes and then remove the ARMChangesRequested label from this PR.